src/ai/backend/manager/models/scaling_group.py

```diff
@@ -315,6 +315,17 @@ class ScalingGroup(graphene.ObjectType):
     scheduler_opts = graphene.JSONString()
     use_host_network = graphene.Boolean()
 
+    # Dynamic fields.
+    agent_count = graphene.Int(description="Added in 24.03.0.")
+
+    async def resolve_agent_count(self, info: graphene.ResolveInfo) -> int:
+        from .agent import Agent
```

Comment:
FYI, to prevent cyclic imports between `agent.py` and `scaling_group.py`, dynamic import used here.